### PR TITLE
Adjust documentation for v11 release

### PR DESCRIPTION
**Pull request by nqb**
# Description
Minor adjustements in documentation to reflect v11 changes

### CODE REVIEW

File: docs/installation/system_requirements.asciidoc [hunk 1]
```diff
@@ -52,10 +52,9 @@ The following provides a list of the minimum server hardware recommendations:
 PacketFence supports the following operating systems on the x86_64 architecture:
 
 [options="compact"]
-* Red Hat Enterprise Linux 7.x Server
-* Community ENTerprise Operating System (CentOS) 7.x
-* Debian 9.0 (Stretch)
+* Red Hat Enterprise Linux 8.x Server
+* Debian 11.0 (Bullseye)
```
Comment (cgx) [inline docs/installation/system_requirements.asciidoc]: We should probably specify `11.x`, as we do for RHEL.

File: docs/common/new_config_files.asciidoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
+After packages upgrades, you should take care to review any changes to configuration files and merge them if required.
```
Comment (extrafu) [inline docs/common/new_config_files.asciidoc]: After packages upgrades, -> Once packages are all upgraded,

File: docs/common/new_config_files.asciidoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+find /usr/local/pf -name \*.rpmnew
+----
+
+.Debian-based systems
+[source,bash]
+----
+find /usr/local/pf -name "*.dpkg-dist"
+----
+
+The list of files returned are the new versions shipped with PacketFence.
```
Comment (extrafu) [inline docs/common/new_config_files.asciidoc:17]: Compare them to your existing version -> Compare them with your installed versions

**Comment by nqb** [comment]
I don't see other things to adjust for v11 release. Please review and let me know if it makes sense for you.

**Comment by extrafu** [comment]
All good for me.

File: docs/installation/installation.asciidoc [hunk 4]
```diff
@@ -51,7 +51,7 @@ Newer version of VMware Player handles the VLAN trunking a lot better.  Having t
 
 === Installing PacketFence on existing Linux
 
-PacketFence provides packages repository for RHEL / CentOS as well as packages repository fo Debian.
+PacketFence provides packages repository for RHEL8 as well as packages repository fo Debian.
```
Comment (cgx) [inline docs/installation/installation.asciidoc]: `s/RHEL8/RHEL 8/`

File: docs/PacketFence_Upgrade_Guide.asciidoc [hunk 5]
```diff
@@ -1628,7 +1637,7 @@ systemctl restart packetfence-mariadb
 
 
 Starting from PacketFence 11.0.0, Debian 9 and CentOS 7 support are dropped in benefit of Debian 11 and RHEL8.
```
Comment (cgx) [inline docs/PacketFence_Upgrade_Guide.asciidoc]: `s/RHEL8/RHEL 8/`

**Comment by nqb** [comment]
All remarks have been taken into account, I squashed commits.

**Comment by extrafu** [comment]
@nqb @fdurand @cgx All good to merge?

**Comment by fdurand** [comment]
ok on my side

**Comment by julsemaan** [comment]
I did test it too and it worked for me

**Comment by cgx** [comment]
let's merge!

**Pull request merged**